pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -89,7 +89,7 @@ func NewNetworkPolicyController(antreaClientGetter agent.AntreaClientProvider,
 	ofClient openflow.Client,
 	ifaceStore interfacestore.InterfaceStore,
 	nodeName string,
-	podUpdates <-chan v1beta2.PodReference,
+	entityUpdates <-chan v1beta2.EntityReference,
 	antreaPolicyEnabled bool,
```

Comment:
can we incorporate  #1753  changes in and add logging and statusMgr knobs? That way we will not need another PR.

Reply:
Again, I’m not sure if separate knobs are needed still… unless you really need to turn off the other Antrea-native policy related features in vmagent. Could you give more insights offline?

Reply:
@Dyanngg as discussed offline, a kind reminder that it is best we can added logging and statusMgr knobs back as described in #1753

Reply:
It's included in the latest commit.